docs/extensions/ScrollViewerExtensions.md

```diff
@@ -55,18 +76,65 @@ class DoubleTopThicknessConverter : IValueConverter
 }
 ```
 
-The converter is used to just bind to top margin, moving only the ScrollBar's top end.
+## MiddleClickScrolling
+
+MiddleClickScrolling allows you to scroll by click middle mouse button (scroll wheel button) and move the pointer of the direction to be scrolled. This extension method can be used directly in `ScrollViewer` or ancestor of `ScrollViewer`.
+
+### Syntax
+
+```xaml
+<!-- Setting MiddleClickScrolling directely for ScrollViewer -->
+<ScrollViewer extensions:ScrollViewerExtensions.EnableMiddleClickScrolling="True">
+    <!-- ScrollViewer Content -->
+</ScrollViewer>
+
+<!-- Setting MiddleClickScrolling fot the ancestor of ScrollViewer -->
+<ListView extensions:ScrollViewerExtensions.EnableMiddleClickScrolling="True">
+    <!-- ListView Item -->
+</ListView>
+```
+
+### Sample Output
+
+![MiddleClickScrolling](../resources/images/Extensions/MiddleClickScrolling.gif)
+
+### Changing Cursor Type
+
+> [!IMPORTANT]
+Resource file must be manually added to change the cursor type when middle click scrolling.
```

Comment:
I'm not sure if I follow what this means to the developer.  Is it required to do these steps to use the extension?